app/src/main/AndroidManifest.xml

```diff
@@ -10,7 +10,9 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.MyAnimeReport">
-        <activity android:name=".activities.SignupActivity"></activity>
+        <activity android:name=".activities.EntryDetailsActivity"></activity>
```

Comment:
nit: You can also remove the closing tag here, and handle similarly to the activities below.